client/src/components/Feed/NestedComments.js

```diff
@@ -74,62 +108,144 @@ const NestedComments = ({ isAuthenticated, comment }) => {
     setReply("");
   };
 
-  const renderReplyInput = showReply ? (
-    <div className="reply-input">
-      {renderAvatar}
-      {isAuthenticated ? (
-        <AutoSize
-          placeholder={"Write a reply..."}
-          onPressEnter={handleReply}
-          onChange={(e) => setReply(e.target.value)}
-          value={reply}
-        />
-      ) : (
-        ""
-      )}
-    </div>
-  ) : (
-    ""
-  );
-
-  const renderReply = () => {
-    const style = showReply ? clickedTextStyle : {};
-    return (
-      <span
-        style={style}
-        onClick={() => setShowReply(!showReply)}
-        key="comment-nested-reply-to"
-      >
-        Reply
-      </span>
-    );
-  };
-  const renderTimeStamp = <span>1w</span>;
+  // const renderReplyInput = showReply ? (
+  //   <div className="reply-input">
+  //     {renderAvatar}
+  //     {isAuthenticated ? (
+  //       <AutoSize
+  //         placeholder={"Write a reply..."}
+  //         onPressEnter={handleReply}
+  //         onChange={(e) => setReply(e.target.value)}
+  //         value={reply}
+  //       />
+  //     ) : (
+  //       ""
+  //     )}
+  //   </div>
+  // ) : (
+  //   ""
+  // );
 
-  //TODO: Add comment replies, like button and number of likes.
-  // const commentActions = [
-  //   renderTimeStamp,
-  //   renderLikeButton(),
-  //   renderReply(),
-  //   renderNumLikes(),
-  //   renderReplyInput,
-  // ];
+  // const renderReply = () => {
+  //   const style = showReply ? clickedTextStyle : {};
+  //   return (
+  //     <span
+  //       style={style}
+  //       onClick={() => setShowReply(!showReply)}
+  //       key="comment-nested-reply-to"
+  //     >
+  //       Reply
+  //     </span>
+  //   );
+  // };
 
   const nestedComments = (comment.children || []).map((comment) => {
     return <NestedComments comment={comment} key={comment._id} />;
   });
 
+  const handleCommentEdit = (e) => {
+    setEditedComment(e.target.value);
+  };
+
+  const handleSubmit = async () => {
+    let response;
+    const postId = comment.postId;
+    const commentId = comment._id;
```

Comment:
can use destructuring on 1 line, also to rename like `const { postId, _id: commentId } = comment`
Also I just  realized, sometimes the `_id` is changed on the backend to `id` but I guess this is not consistent for every response